docs/tutorials/using_apis/connectivity/light-control.md

```diff
@@ -0,0 +1,229 @@
+|-----------|-------------------------|------------------------------------|
+|Mesh router (default) | lowpan_router | NET_6LOWPAN_ROUTER |
+|Nonrouting device | lowpan_host | NET_6LOWPAN_HOST |
+
+#### Thread
+
+**nsapi.default-mesh-type: THREAD**
+
+|Device role|`nanostack.configuration` value|`mbed-mesh-api.thread-device-type` value|
+|-----------|-------------------------|------------------------------------|
+|Mesh router (default) | thread_router | MESH_DEVICE_TYPE_THREAD_ROUTER |
+|Nonrouting device | thread_end_device | MESH_DEVICE_TYPE_THREAD_SLEEPY_END_DEVICE |
+
+##### Thread commissioning
+
+By default, the Thread application uses the static network link configuration defined in the [mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed_lib.json).
+If you want to commission a Thread device, see [how to commission a Thread device in practice](../reference/mesh-tech.html#thread-commissioning).
+
+The Thread stack learns the network settings from the commissioning process and saves them to RAM memory. Therefore, the learned network settings are lost when you restart the device next time. To prevent re-commissioning, you can save the Thread configuration settings to an SD card as follows (only in `K64F`):
+
+- Change `storage-device` to `MESH_NVM_SD_CARD` in the `./configs/mesh_thread.json` file.
+- Enable commissioning as descibed in the referred instructions.
+- Compile and program the application.
+- Insert an erased or FAT-formatted SD card to the `K64F` memory card slot. The application will initialize the SD card with the appropriate file system on first use.
+- Commission the device to the Thread network.
+- When you restart the device next time, the device reads the Thread configuration settings from the SD card and tries to attach to an existing network.
+
+### Requirements for hardware
+
+The networking stack in this example requires TLS functionality to be enabled on Mbed TLS. On devices where hardware entropy is not present, TLS is disabled by default. This results in compile time failures or linking failures.
+
+To learn why entropy is required, read the [TLS porting guide](../reference/mbed-tls-entropy.html).
+
+See [Notes on different hardware](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/Hardware.md) for known combinations of development boards and RF shields that have been tested.
+
+You also need to check how LEDs and buttons are configured for your hardware, and update the `.json` file accordingly.
+
+### Changing the radio driver
+
+To run a 6LoWPAN-ND network, you need a working RF driver for Nanostack. This example uses the Atmel AT86RF233 by default.
+
+To change the RF driver, modify the `mbed_app.json` file by setting preferred RF driver `provide_default` value to true, For example, to use MCR20a RF driver: 
+
+```json
+"atmel-rf.provide-default": false,
+"mcr20a.provide-default": true,
```

Comment:
I _think_ the Travis check is complaining that something is wrong with this JSON.

If I had to guess, the trailing comma on this line implies another key/value pair, which doesn't appear.

@geky Can you confirm?

Reply:
Yep, json is pedantic as usual.